### DISCUSSION TITLE
I Keep Getting My Application for the Student Developer Pack Rejected

### DISCUSSION THREAD

**Discussion by oarrambide1217** [community/community#17775] [GitHub Education]
Hi. I have been trying to get my application for the student developer pack to go through but after several attempts it keeps getting rejected. I have uploaded my tution receipts from my school bursar's office, I have uploaded my class schedule, and my school provided  student bio. My last application was rejected on the basis that my browser says that I am in Mexico while my school is in OK, USA. It is true that I live in Mexico, but I am an online student and all my documents state that I am an online student. I honestly don't know what to add anymore. If anyone can help, that would be amazing.

**Comment by mlarasusan** [comment]
Hello @oarrambide1217 👋 I have some information here that can help you [✨](https://emojipedia.org/sparkles/)

👉 Check [this](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) to learn how to get the GitHub Student Developer Pack without a student ID
👉 Also, check the reviews of these students: [1](https://github.com/orgs/github-community/discussions/31#discussioncomment-2561585); [2](https://github.com/orgs/github-community/discussions/16); [3](https://github.com/orgs/github-community/discussions/65) & [4](https://github.com/orgs/github-community/discussions/67) - Looks like the last two had the same issue as you
👉 And here is the [link](https://support.github.com/contact/education) to contact our support team
👉 Student Developer Pack Application & [FAQs](https://github.com/orgs/github-community/discussions/9)
